charts/bluehub/v0.1.0/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+name: bluehub
+home: https://thebluespire.com/kafka-distribution
+icon: file://./bluehub.png
```

Comment:
Move the icon one level up to the `bluehub` directory and fix reference here. This way you can use the same icon for future chart versions.

Reply:
changes made as suggested